app/src/main/res/layout/activity_sign_in.xml

```diff
@@ -38,7 +97,7 @@
         android:layout_gravity="end"
         android:onClick="showPasswordCheck"
         android:text="@string/Show_Password" />
-
+    -->
```

Comment:
I actually don't know what this is for. Perhaps delete it? If it is important it can stay.